pkg/controller/perconaservermongodb/smart.go

```diff
@@ -58,12 +59,21 @@ func (r *ReconcilePerconaServerMongoDB) smartUpdate(cr *api.PerconaServerMongoDB
 		return nil
 	}
 
-	ok, err := r.isBackupRunning(cr)
+	isBlocked, err := r.isBackupRunning(cr)
 	if err != nil {
-		return fmt.Errorf("failed to check active backups: %v", err)
+		return errors.Wrap(err, "failed to check active backups")
 	}
-	if ok {
-		log.Info("can't start 'SmartUpdate': waiting for running backups finished")
+	if isBlocked {
+		log.Info("can't start 'SmartUpdate': waiting for running backups to be finished")
+		return nil
+	}
+
+	isBlocked, err = backup.HasActiveJobs(r.client, cr, backup.Job{}, backup.NotPITRLock)
```

Comment:
to avoid misleading worth to rename to smth like `hasActiveJobs`

Reply:
updated in 2ab5eebbccd167792aea4b69b5fae6680f19e217